src/components/svgIcons/societyIcons/Istelle.jsx

```diff
@@ -1,15 +1,40 @@
 import React from 'react';
 
+const svgPointA = 'M239,317.54c4.92,0.32,9.84.63,14.76,1q8.71,0.62,17.42,1.3c3.31,0.26,6.63.47,9.93,0.88,5.52,0.68,9,' +
```

Comment:
This is too much work for a minor thing. You could disable the line-length rule for this component and others like it. Nobody ever reads SVG anyway.

Reply:
Now the work is done.